tests/devapps/XForms/XForms.Android/XForms.Android.csproj

```diff
@@ -1,139 +1,159 @@
-    <AndroidResource Include="Resources\drawable-xxhdpi\icon.png" />
-  </ItemGroup>
-  <ItemGroup>
-    <AndroidResource Include="Resources\layout\Tabbar.axml" />
-    <AndroidResource Include="Resources\layout\Toolbar.axml" />
-    <AndroidResource Include="Resources\values\styles.xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="AsyncUsageAnalyzers">
-      <Version>1.0.0-alpha003</Version>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3">
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Xamarin.Forms" Version="3.5.0.129452" />
-    <PackageReference Include="Xamarin.Android.Support.V4" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Compat" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Design" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Core.UI" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.v7.CardView" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Annotations" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.v7.MediaRouter" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.v7.Palette" Version="28.0.0.3" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="StrongNamer">
-      <Version>0.0.8</Version>
-    </PackageReference>
-    <ProjectReference Include="..\..\..\..\src\client\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj">
-      <Project>{3433eb33-114a-4db7-bc57-14f17f55da3c}</Project>
-      <Name>Microsoft.Identity.Client</Name>
-    </ProjectReference>
-    <ProjectReference Condition="'$(ARIA_TELEMETRY_ENABLED)' != ''" Include="..\..\AriaTelemetryProvider\AriaTelemetryProvider.csproj">
-      <Project>{2ae44538-8e72-4d43-9cd1-b26c3a657814}</Project>
-      <Name>AriaTelemetryProvider</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\XForms\XForms.csproj">
-      <Project>{a00ac06a-694f-4fea-999f-81e753fa7c28}</Project>
-      <Name>XForms</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-  <ProjectExtensions>
-    <VisualStudio>
-      <UserProperties XamarinHotReloadXFormsNugetUpgradeInfoBarXFormsAndroidHideInfoBar="True" />
-    </VisualStudio>
-  </ProjectExtensions>
-</Project>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Are these changes (and the ones in the solution file, related to that tab/spaces formatting issue? I think it would be helpful to fix this to make it easier to see what changes are in these files.

Reply:
It looks like the tab/space issue that comes from UTF-8 BOM. Fixed